autoload/lsp.vim

```diff
@@ -136,6 +136,7 @@ function! s:register_events() abort
         autocmd BufWritePost * call s:on_text_document_did_save()
         autocmd BufWinLeave * call s:on_text_document_did_close()
         autocmd InsertLeave * call s:on_text_document_did_change()
+        autocmd TextChanged * call s:on_text_document_did_change()
```

Comment:
we need to debouce `TextChanged` else did change notification will be called a lot. When we have debounce we could add this. Else can we add this behind a flag?